src/pages/workspace/reportFields/ReportFieldsValueSettingsPage.tsx

```diff
@@ -58,18 +59,17 @@ function ReportFieldsValueSettingsPage({
     }, [formDraft?.disabledListValues, formDraft?.listValues, policy?.fieldList, reportFieldID, valueIndex]);
 
     const hasAccountingConnections = PolicyUtils.hasAccountingConnections(policy);
+    const oldValueName = usePrevious(currentValueName);
```

Comment:
@shubham1206agra i think we don't need to do that

Reply:
Do you mean we should update it to
```
const oldValueName = usePrevious(currentValueName ?? oldValueName);
```

If that's the case, it could cause an infinite loop or the value might not be updated correctly.

Reply:
@shubham1206agra i updated